features.json

```diff
@@ -126,6 +126,7 @@
 				"exceptions": "100",
 				"extendedConst": "112",
 				"gc": "120",
+				"jspi": ["flag", "Enabled in Nightly, unavailable in Beta/Release"],
```

Comment:
"Preferenced off" (?) would be a better wording -- a user must change the "javascript.options.wasm_js_promise_integration" flags.

The "unavailable in Beta/Release" is correct.